src/app/components/mnoe-api/provisioning.svc.coffee

```diff
@@ -170,11 +170,11 @@ angular.module 'mnoEnterpriseAngular'
       )
       return deferred.promise
 
-    @emptySubscriptions = () ->
+    @emptyCartSubscriptions = () ->
       _self.cartSubscriptionsPromise = null
 
-    @refreshSubscriptions = ->
-      _self.emptySubscriptions()
-      _self.getSubscriptions({ where: {staged_subscriptions: true } })
+    @refreshCartSubscriptions = ->
+      _self.emptyCartSubscriptions()
+      _self.getSubscriptions({ where: {subscription_status_in: 'staged' } })
```

Comment:
@MAhsenArif Why was this changed?

Reply:
@iseessel Wrong param was passing here. `staged_subscriptions` filter got remade to `subscription_status_in` [here](https://github.com/maestrano/maestrano-hub/blob/develop/app/resources/api/mnoe/v2/subscription_resource.rb#L39)

Reply:
👍 Thanks!